text/machine_comprehension/roberta/README.md

```diff
@@ -0,0 +1,67 @@
+# RoBERTa
+
+## Use cases
+This transformer-based sequence clssification model that predicts sentiment based on given input text.   
+
+## Description
+RoBERTa builds on BERT’s language masking strategy and modifies key hyperparameters in BERT, including removing BERT’s next-sentence pretraining objective, and training with much larger mini-batches and learning rates. RoBERTa was also trained on an order of magnitude more data than BERT, for a longer amount of time. This allows RoBERTa representations to generalize even better to downstream tasks compared to BERT.
+
+## Model
+
+ |Model        |Download  |Download (with sample test data)| ONNX version |Opset version|Accuracy|
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
+|RoBERTa| [249 MB](model/roberta-9.onnx) |  [231 MB](model/roberta-9.tar.gz) |  1.4 | 9|mAP of [0.0183](https://docs.google.com/spreadsheets/d/1sryqufw2D0XlUH4sq3e9Wnxu5EAQkaohzrJbd5HdQ_w/edit#gid=0)|
```

Comment:
0.0183 is latency (in seconds) instead of accuracy.

Reply:
Fixed accuracy score and provided a link for more information.